src/main/java/school/faang/stream4/alchemylevelingup/PotionGathering.java

```diff
@@ -0,0 +1,64 @@
+package school.faang.stream4.alchemylevelingup;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.function.IntFunction;
+
+@Slf4j
+public class PotionGathering {
+    private static ExecutorService service;
```

Comment:
если уж static, то можно сделать полноценную константу:
```suggestion
    private static final ExecutorService SERVICE;
```

Reply:
Вы там определитесь, мне в другой задаче Юсупов на константный пулл потоков сказал, что не стоит делать его константным

Reply:
Учитывая концепцию программы, где у тебя 1 класс и ты полностью управляешь жизненным циклом ExecutorService, его вполне можно делать константным, инициализируя на старте программы и "убивая" в конце. Всё зависит от сложности среды (приложения), в частности, в контексте текущих задач его везде можно делать константным, в иных ситуациях будет использоваться поход DependencyInjection